ipaserver/plugins/trust.py

```diff
@@ -446,8 +446,13 @@ def fetch_trusted_domains_over_dbus(myapi, *keys, **options):
         fetch_domains_method = intf.get_dbus_method(
                 'fetch_domains',
                 dbus_interface=DBUS_IFACE_TRUST)
+        # Oddjobd D-BUS method definition only accepts fixed number
+        # of arguments on the command line. Thus, we need to pass
+        # remaining ones as ''. There are 30 slots to allow for extension.
```

Comment:
Perhaps mention that the limit is specified in ``oddjobd-ipa-trust.conf``.

Reply:
Added.